votingpool/db.go

```diff
@@ -116,17 +185,18 @@ func putSeriesRow(tx walletdb.Tx, votingPoolID []byte, ID uint32, row *dbSeriesR
 	bucket, err := tx.RootBucket().CreateBucketIfNotExists(votingPoolID)
 	if err != nil {
 		str := fmt.Sprintf("cannot create bucket %v", votingPoolID)
-		return managerError(waddrmgr.ErrDatabase, str, err)
+		return newError(ErrDatabase, str, err)
 	}
+	bucket = bucket.Bucket(seriesBucketName)
 	serialized, err := serializeSeriesRow(row)
 	if err != nil {
 		str := fmt.Sprintf("cannot serialize series %v", row)
-		return managerError(waddrmgr.ErrSeriesStorage, str, err)
+		return newError(ErrSeriesStorage, str, err)
```

Comment:
Yeah, that's because we're returning ErrSeriesStorage when we get a bolt error (in which case we use that as the underlying error) but also when we fail to serialize the series data (in which case we have no underlying error). Maybe we should use it only for the latter (and possibly rename it to ErrSeriesSerialization), and use ErrDatabase for the former?